packages/muelu/src/Utils/MueLu_AvatarInterface.cpp

```diff
@@ -327,13 +331,16 @@ void AvatarInterface::SetMueLuParameters(const Teuchos::ParameterList & problemF
     }
     else {
       // As a placeholder, we'll choose the first acceptable combination.  Later we can do something smarter
+      //TODO: Look at other options to choose best possible combination
       chosen_option_id = acceptableCombos[0];
     }
 
     // Generate the parameterList from the chosen option
     GenerateMueLuParametersFromIndex(chosen_option_id,avatarParams);
-  }
 
+    // Cleanup
```

Comment:
It might be good if the interface to Avatar had a separate `avatar_free_test_result` kind of function, so that users don't need to know whether the result of `avatar_test` needs `free` or some other deallocation function.